dev_notes/dev-quick-start.md

```diff
@@ -49,15 +49,17 @@ kubectl config set-context --current --namespace=mongodb
 
 #### create a config file for the dev environment
 ```bash
+mkdir -p ~/.community-operator-dev
 cat > ~/.community-operator-dev/config.json << EOL
 {
     "namespace": "default",
     "repo_url": "localhost:5000",
-    "operator_image": "mongodb-kubernetes-operator",
+    "operator_image": "community-operator-dev",
     "e2e_image": "e2e",
     "prestop_hook_image": "prehook",
     "version_upgrade_hook_image": "community-operator-version-upgrade-post-start-hook",
-    "readiness_probe_image": "mongodb-kubernetes-readinessprobe"
+    "readiness_probe_image": "mongodb-kubernetes-readinessprobe",
+    "s3_bucket": ""
```

Comment:
otherwise the following exception is raised
```
image_build_start: e2e
[e2e-template/dockerfile_template] stage-started e2e-template: 1/2
[e2e-template/dockerfile_template] dockerfile-save-location: scripts/dev/templates/Dockerfile.ubi-fb70c354-18d1-4b7e-ab3b-c5844b5645ed
[e2e-build/docker_build] stage-started e2e-build: 2/2
[e2e-build/docker_build] docker-image-push: localhost:5000/community-operator-e2e:fb70c354-18d1-4b7e-ab3b-c5844b5645ed
[e2e-build/docker_build] docker-image-push: localhost:5000/community-operator-e2e:latest
python pipeline.py --image-name agent-ubuntu
Traceback (most recent call last):
  File "/Users/alisovenko/workspace/mongodb-kubernetes-operator/pipeline.py", line 179, in <module>
    sys.exit(main())
  File "/Users/alisovenko/workspace/mongodb-kubernetes-operator/pipeline.py", line 174, in main
    image_build_function(config)
  File "/Users/alisovenko/workspace/mongodb-kubernetes-operator/pipeline.py", line 57, in build_agent_image_ubuntu
    args = _build_agent_args(config)
  File "/Users/alisovenko/workspace/mongodb-kubernetes-operator/pipeline.py", line 39, in _build_agent_args
    "s3_bucket": config.s3_bucket,
  File "/Users/alisovenko/workspace/mongodb-kubernetes-operator/scripts/dev/dev_config.py", line 55, in s3_bucket
    return self._config["s3_bucket"]
KeyError: 's3_bucket'
make: *** [agent-image] Error 1
```